_sass/nav.scss

```diff
@@ -0,0 +1,10 @@
+@import "colors.scss";
```

Comment:
Basic header... doesn't really need to be there but figured it'd be a good example. 

![image](https://user-images.githubusercontent.com/117696/78908454-efe7f400-7a36-11ea-9e32-91a174516219.png)